components/dashboard/package.json

```diff
@@ -5,6 +5,7 @@
   "private": true,
   "dependencies": {
     "@gitpod/gitpod-protocol": "0.1.5",
+    "moment": "^2.29.1",
```

Comment:
> Moment.js is a legacy project, now in maintenance mode. In most cases, you should choose a different library.

nit: it would be great to find an alternative to format relative time. this is quite a balloon in the resulting bundle.

Reply:
if it's used just to format relative time, maybe it's good to use https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/RelativeTimeFormat, and there is a polyfill available, too.